### PR TITLE
[ List.All ] points-to relation and props of updateAt modelled after Vec

### PR DESCRIPTION
**Pull request by andreasabel**
Made updateAt of List.All analogous to updateAt of Data.Vec.

### CODE REVIEW

File: src/Data/List/Relation/Unary/All/Properties.agda [hunk 1]
```diff
@@ -115,68 +141,196 @@ module _ {_~_ : REL (List A) B ℓ} where
     (x~y ∷ (All.map All.head pxss)) ∷
     All-swap (x~ys ∷ (All.map All.tail pxss))
 
+------------------------------------------------------------------------
+-- Defining properties of lookup and _[_]=_
+--
+-- pxs [ i ]= px  if and only if  lookup pxs i = px.
+
+module _ {a p} {A : Set a} {P : A → Set p} where
+
```
Comment (gallais) [inline src/Data/List/Relation/Unary/All/Properties.agda]: We have `variable`s for `a` and `A` so we can drop some of these parameters.

Comment (MatthewDaggitt) [reply src/Data/List/Relation/Unary/All/Properties.agda]: and `p`

**Comment by MatthewDaggitt** [comment]
Looks good to me.

**Comment by MatthewDaggitt** [comment]
@andreasabel just thought I should mention that unless there's an urgent reason to merge it in quickly or the change is purely cosmetic, we usually leave PRs up for a week or so to give everyone the chance to provide feedback. Stops people from having to log on everyday! No worries in this case but just a general heads-up for future PRs :+1: Actually I should probably write this down somewhere :smile:

**Comment by andreasabel** [comment]
Ok, thanks for letting me know, shall observe this in the future.

**Pull request merged**